.codedeploy/scripts/add_node_specific_config

```diff
@@ -23,10 +23,10 @@ FLAGS=
 
 case "$NAME" in
 	vln)
-		FLAGS="--ws-port 80 --rpc-port 8080 --rpc-external --ws-external --rpc-cors all --node-key $VLN1_KEY"
+		FLAGS="--collator --ws-port 80 --rpc-port 8080 --rpc-external --ws-external --rpc-cors all -- --execution wasm --chain rococo"
```

Comment:
These are the flags only for the full node, so no `--collator`. I haven't run a full node of a parachain so I don't know if it needs anything extra but I assume nothing should change? We still need the full node to have a well known PeerId to be the boot node of the collators so that part has to stay and I don't know if it needs the relaychain part but I think it doesn't right?